proposals/proposal_template.md

```diff
@@ -0,0 +1,140 @@
+# Proposal: &lt;API>
+
+**Summary**
+
+An API proposal for your new API.  Describe in **one sentence** what it will be
+used for.
+
+**Document Metadata**
+
+**Author:** &lt;your Github handle>
+
+**Sponsoring Browser:** The browser vendor committed to implementing this API
+if approved.
+
+**Contributors:** &lt;other contributors @>
```

Comment:
What is the `@` referring to? Would be nice to suggest the Github handle if any, or any other form of identification (whether an email, or a profile on the codereview site).

Reply:
Rephrased.